### PR TITLE
Create README

### PR DESCRIPTION
**Pull request by TycheLaughs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# sonifyme
+App that matches people with pieces from within their favorite musician's body of work
```
Comment (krsween) [inline README.md]: @TycheLaughs this line's a bit confusing. Can you rephrase that for me? Thanks!

**Comment by TycheLaughs** [comment]
>App that matches people with songs and albums from their favorite musical artist or band that they personify

This better?

**Comment by krsween** [comment]
@TycheLaughs perfect! Thanks for making those tweaks.
